<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>880</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="2">
    <widget class="QPushButton" name="save">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Article information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Keywords:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2" colspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>170</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QLCDNumber" name="year">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Abstract:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="previous">
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="discard">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Discard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2" rowspan="4" colspan="3">
       <widget class="QPlainTextEdit" name="keywords">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>Keywords
</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>170</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" rowspan="5" colspan="2">
       <widget class="QPlainTextEdit" name="abstract_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>Abstract</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QPlainTextEdit" name="title">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="12" column="4">
       <widget class="QLCDNumber" name="number">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2" colspan="3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Web browse</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="google">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image :url(\&quot;edg.png\&quot;);
background-repeat: no-repeat;
background-position: center;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sciHub">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image :url(\&quot;sci-hub.png\&quot;);
background-repeat: no-repeat;
background-position: center;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="filename">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="3" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Comments</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Control:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QComboBox" name="k2">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Methodology:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="4">
       <widget class="QTextEdit" name="conclusions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Keyword1:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="4">
       <widget class="QTextEdit" name="results">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Keyword2</string>
        </property>
       </widget>
      </item>
      <item row="12" column="4">
       <widget class="QPushButton" name="openFile">
        <property name="text">
         <string>Open file</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="3">
       <widget class="QLineEdit" name="pkl_name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Results:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QComboBox" name="k1">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Optimization:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="4">
       <widget class="QTextEdit" name="methodology">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QComboBox" name="type">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QComboBox" name="optimization">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QComboBox" name="control">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Conclusions:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="browse">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="directory">
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image :url(\&quot;folder.jpg\&quot;);
background-repeat: no-repeat;
background-position: center;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
